app/assets/json/translations/locale-en.json

```diff
@@ -442,6 +442,11 @@
         "APPLICANT_SEPARATE_ADDRESS": "Send my mail to a different address",
         "APPLICANT_SEXUAL_ORIENTATION": "How do you describe your sexual orientation or sexual identity?",
         "APPLICATION_CONTACT_DETAILS": "Application Contact Details",
+        "APPLICATION_IS_FIRST_TIME_HOMEBUYER": "I have not owned property within the past three years.",
+        "APPLICATION_HAS_COMPLETED_HOMEBUYER_EDUCATION": "I have completed homebuyers’ education.",
+        "APPLICATION_VISIT_HOMEOWNERSHIP_CALENDAR": "If you haven't done this yet, visit the <a href='{{url}}' target='_blank'>HomeownershipSF calendar</a> to attend a workshop near you.",
+        "APPLICATION_HAS_LOAN_PRE_APPROVAL": "I am pre-approved for a loan by a MOHCD-approved lender.",
+        "APPLICATION_VISIT_MOHCD": "If you haven't done this yet, visit the <a target='{{url}}'>MOHCD website</a> to find a list of approved lenders",
```

Comment:
This link will be off-the application too so we should open it in a new tab with target='_blank'